src/main/python/rlbot/utils/structures/game_data_struct.py

```diff
@@ -145,34 +154,52 @@ class ByteBuffer(Struct):
 # This negates all x and y values for balls and cars and rotates yaw 180 degrees.
 def rotate_game_tick_packet_boost_omitted(game_tick_packet):
     # Negate all x,y values for ball
-    game_tick_packet.game_ball.physics.location.x = -1 * game_tick_packet.game_ball.physics.location.x
-    game_tick_packet.game_ball.physics.location.y = -1 * game_tick_packet.game_ball.physics.location.y
-    game_tick_packet.game_ball.physics.velocity.x = -1 * game_tick_packet.game_ball.physics.velocity.x
-    game_tick_packet.game_ball.physics.velocity.y = -1 * game_tick_packet.game_ball.physics.velocity.y
+    game_tick_packet.game_ball.physics.location.x = - \
+        1 * game_tick_packet.game_ball.physics.location.x
```

Comment:
The  `-` being on a different line to the `1` makes it easy to mis-read by missing the negation.
Put the `-` on the new line too.

Reply:
This was the fault of my auto-formatted :(